source/index.md

```diff
@@ -16,153 +10,204 @@ includes:
+    
+* Update
 
-<aside class="notice">
-You must replace <code>meowmeowmeow</code> with your personal API key.
-</aside>
+ You also can update a telephone using a command line, with the id of the telephone, or accessing a route, both cases are described below.
 
-# Kittens
+ Route: `PATCH "/api/v1/manager/client-telephones/:id"`
 
-## Get All Kittens
+ Command Line: `ruby bin/horus-cli update telephones "id" '"country-code":"1", "number":"789890888"'`
 
-```ruby
-require 'kittn'
+* List
 
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-api.kittens.get
-```
+ You also can list your telephones using a command line or accessing a route, both cases are described below.
 
-```python
-import kittn
+ Route: `GET "/api/v1/manager/client-telephones"`
 
-api = kittn.authorize('meowmeowmeow')
-api.kittens.get()
-```
+ Command Line: `ruby bin/horus-cli list telephones`
 
-```shell
-curl "http://example.com/api/kittens"
-  -H "Authorization: meowmeowmeow"
-```
+* Show
 
-> The above command returns JSON structured like this:
+ You also can show a telephone using a command line, with the id of the telephone, or accessing a route, both cases are described below.
 
-```json
-[
-  {
-    "id": 1,
-    "name": "Fluffums",
-    "breed": "calico",
-    "fluffiness": 6,
-    "cuteness": 7
-  },
-  {
-    "id": 2,
-    "name": "Isis",
-    "breed": "unknown",
-    "fluffiness": 5,
-    "cuteness": 10
-  }
-]
-```
+ Route: `GET "/api/v1/manager/client-telephones/:id"`
 
-This endpoint retrieves all kittens.
+ Command Line: `ruby bin/horus-cli show telephones "id"`
 
-### HTTP Request
+## Address
 
-`GET http://example.com/api/kittens`
+ Once authenticated, except a user, you can create new address for a specific resource, edit, list and show it through the following command lines. A resource can only have an address.
 
-### Query Parameters
+* Create
 
-Parameter | Default | Description
---------- | ------- | -----------
-include_cats | false | If set to true, the result will also include cats.
-available | true | If set to false, the result will include kittens that have already been adopted.
+ You can create an address using a command line, with the id of the resource to which it belongs, or accessing a route, both cases are described below.
 
-<aside class="success">
-Remember — a happy kitten is an authenticated kitten!
-</aside>
+ Route: `POST "/api/v1/manager/client-addresses"`
 
-## Get a Specific Kitten
+ Command Line: `ruby bin/horus-cli create addresses '"street" => "Rua", "number" =>  "55", "complement" => "Casa", "zipcode" => "37500344", "neighborhood" => "Bairro", "city" => "Cidade", "state" => "Estado", "country" => "Pais"' clients "id"`
 
-```ruby
-require 'kittn'
+* Update
 
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-api.kittens.get(2)
-```
+ You also can update an address using a command line, with the id of the assress, or accessing a route, both cases are described below.
```

Comment:
Correcting: 'with the id of the assress' - Should be "with the id of the address"